### PR TITLE
added some specs

### PR DESCRIPTION
**Pull request by timur-rimsky**

### CODE REVIEW

File: spec/mxmnd_spec.rb [hunk 1]
```diff
@@ -5,3 +5,49 @@
     expect { Mxmnd::city(nil) }.to raise_error Mxmnd::BadRequest
   end
 end
+
+describe 'Mxmnd.city response' do
```
Comment (Valve) [inline spec/mxmnd_spec.rb:9]: Ruby uses 2 spaces for indentation, you seem to be using more. 
Please install Rubocop locally and run your code through it to see some of the styling issues

File: spec/mxmnd_spec.rb [hunk 2]
```diff
@@ -5,3 +5,49 @@
...
+	before {
+		url = 'https://geoip.maxmind.com/geoip/v2.1/city/45.17.22.11'
+		path = './spec/tmp/mxmnd_res.txt'
+# i've changed "Content-Length" property of recorded response from 1476 to 2515
```
Comment (Valve) [inline spec/mxmnd_spec.rb]: whoa this is weird, why aren't you indenting comments at the same level?

File: spec/tmp/mxmnd_res.txt [hunk 3]
```diff
@@ -0,0 +1,97 @@
+HTTP/1.1 200 OK
```
Comment (Valve) [inline spec/tmp/mxmnd_res.txt]: this file should be in spec/fixtures

**Review by Valve — CHANGES_REQUESTED** [review]
Please fix the formatting first. 
Also the spec actually failed. So you added a spec that failed, see Travis output. Looks like you didn't commit the changes in the Gemfile (or gemspec).

**Comment by Valve** [comment]
@timur-rimsky why did you close the PR?

**Comment by timur-rimsky** [comment]
потому что не получалось сделать новый пул реквест

**Pull request closed without merging**